nixos/tests/podman-compat.nix

```diff
@@ -0,0 +1,81 @@
+/*
+  This test runs podman as a backend for the Docker CLI.
+ */
+import ./make-test-python.nix (
+  { pkgs, lib, ... }:
+  {
+    name = "podman";
+    meta = {
+      maintainers = lib.teams.podman.members;
+    };
+
+    nodes = {
+      podman =
+        { pkgs, ... }:
+        {
+          virtualisation.podman.enable = true;
+          virtualisation.podman.dockerSocket.enable = true;
```

Comment:
Apologies if I was unclear but I would like this moved into `nixos/tests/podman.nix`.

I don't think testing just the local socket needs a separate test.

Reply:
Done.